rails/spec/models/password_spec.rb

```diff
@@ -18,6 +18,7 @@
 
     it 'does  create with proper email' do
       password.email = 'abc@example.com'
+      password.user = user
```

Comment:
I am not sure why this test passed before, as the model has a validation on the user field:

password.rb:9
`  validates_presence_of :email, :user `